deploy/central/tasks-chart/templates/task-ods-build-typescript.yaml

```diff
@@ -36,10 +36,14 @@ spec:
       description: Whether to skip the SonarQube analysis or not.
       type: string
       default: "false"
+    - name: node-version
+      description: "Node.js version to use - supported versions: 16"
+      type: string
+      default: "16"
   steps:
     - name: build-typescript
-      # Image is built from build/package/Dockerfile.typescript-toolset.
-      image: '{{.Values.registry}}/{{.Values.namespace}}/ods-typescript-toolset:{{.Values.imageTag}}'
+      # Image is built from build/package/Dockerfile.node-xx-typescript-toolset.
```

Comment:
I like this approach. Swapping the image while keeping one task seems like the best of both worlds. Really nice.